bridge-history-api/internal/logic/l2_fetcher.go

```diff
@@ -85,7 +85,12 @@ func NewL2FetcherLogic(cfg *config.FetcherConfig, db *gorm.DB, client *ethclient
 
 	if common.HexToAddress(cfg.LIDOGatewayAddr) != (common.Address{}) {
 		addressList = append(addressList, common.HexToAddress(cfg.LIDOGatewayAddr))
-		gatewayList = append(gatewayList, common.HexToAddress(cfg.USDCGatewayAddr))
+		gatewayList = append(gatewayList, common.HexToAddress(cfg.LIDOGatewayAddr))
```

Comment:
This didn't break anything before?

Reply:
Luckily it didn't, nobody sent a reverted transaction to this gateway yet.
The `gatewayList` is only used to track failed transactions.
https://scrollscan.com/address/0x8aE8f22226B9d789A36AC81474e633f8bE2856c9